server/src/test/java/org/elasticsearch/action/search/SearchResponseMergerTests.java

```diff
@@ -452,6 +455,49 @@ public void testMergeCompletionSuggestionsTieBreak() throws InterruptedException
         }
     }
 
+    // Test merging results where one result has a raw format, for instance if
+    // searching over multiple indexes where the field isn't mapped in all indexes.
+    public void testMergeEmptyFormat() throws InterruptedException {
```

Comment:
If one field isn't mapped I think we use the mapped field as the "lead" for the merge. Does this happen in production?

Reply:
Bit of an off the cuff read, so I could be wrong, but I'm pretty sure we don't do that.  We've had problems around this with terms over IP fields that were unmapped in some indices, where the unmapped ones came back as keywords (the default).  Often we use `value_type` to nudge it, but I don't think anything in the merge picks the right one naturally.

Reply:
It's quite possible for two aggs to have a different type and for that to have an empty type. But the "real" one will be first. Sorry for the confusion.